check-diff.sh

```diff
@@ -844,7 +844,9 @@ function lint_php_files {
 			echo "## PHP_CodeSniffer"
 			cd "$LINTING_DIRECTORY"
 			if ! cat "$TEMP_DIRECTORY/paths-scope-php" | remove_diff_range | xargs phpcs -s --report-emacs="$TEMP_DIRECTORY/phpcs-report" --standard="$( if [ ! -z "$PHPCS_RULESET_FILE" ]; then echo "$PHPCS_RULESET_FILE"; else echo "$WPCS_STANDARD"; fi )" $( if [ -n "$PHPCS_IGNORE" ]; then echo --ignore="$PHPCS_IGNORE"; fi ); then
-				if [ "$CHECK_SCOPE" == 'patches' ]; then
+				if [ ! -s "$TEMP_DIRECTORY/phpcs-report" ]; then
```

Comment:
This should work. But also what about looking at`$?`? I assume that it would be the exit code of `phpcs`, I'm guessing `254`. If it outputs a specific exit code when a fatal error happens, then that could be checked for here instead:

```bash
php_exit_code=$?
if [ "$php_exit_code" -gt 200 ];then
    return $php_exit_code
fi
```